ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/simple.html

```diff
@@ -0,0 +1,35 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<template data-sly-template.simple="${ @ container}">
+    <sly data-sly-test="${wcmmode.edit}" data-sly-use.allowed="com.day.cq.wcm.foundation.AllowedComponentList"></sly>
+    <div data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+         data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+         id="${container.id}"
+         data-cmp-is="adaptiveFormPanel"
+         data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
+         class="cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}"
+         aria-label="${container.accessibilityLabel}"
+         role="${container.roleAttribute}"
```

Comment:
what kind of roles a panel can have ?

Reply:
inherited from sites component, they haven't documented it, but for container it can align to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#1._document_structure_roles